### ISSUE TITLE
[Bug] Setting Shell.PresentationMode=ModalAnimated does not hide NavigationBar on ModalPage

### ISSUE THREAD

**Issue by HobDev** [xamarin/Xamarin.Forms#10309]
### Steps to Reproduce

1. 
2. 
3. 

### Basic Information

- Version with issue:
- Last known good version:
- IDE:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

### Reproduction Link
[ShellNavigationSample.zip](https://github.com/xamarin/Xamarin.Forms/files/4466239/ShellNavigationSample.zip)

**Comment by jsuarezruiz** [comment]
@HobDev If you use **GoToAsync** (in combination with Modal PresentationMode) it will push modally so the page will be modal. In other case, for example, using PushAsync will not have any effect.

**Comment by HobDev** [comment]
@jsuarezruiz I want to present the LoginPage as Modal. LoginPage is first and current item of the shell on app startup. So, I don't have to use GoToAsync or PushModalAsync. I just need to set PresentationMode to Modal.

**Comment by HobDev** [comment]
I didn't tested the Modal Presentation with other pages in the Shell.

**Comment by jsuarezruiz** [comment]
@PureWeen I would like to have your opinion in this case.

**Comment by HobDev** [comment]
Any updates on this

**Comment by PureWeen** [comment]
@HobDev  If I'm understanding your scenario here you're wanting the LoginPage to be specified in the AppShell file. 

It'll be the first page that loads and it'll load modally?

Currently the presentation mode only works for pages you are pushing onto the stack but just making sure I'm following your sample

**Comment by HobDev** [comment]
@PureWeen  Exactly, I want to load LoginPage modally and it is specified in AppShell.

 Additionally I am curious to know the difference between ModalPage and ContentPage with hidden NavBar. It seems to me that ModalPage is also hiding the NavBar. I didn't notice any other difference.
